CMakeLists.txt

```diff
@@ -14,17 +14,30 @@ ELSE(OPENMP_FOUND)
   message("ERROR: OpenMP could not be found.")
 ENDIF(OPENMP_FOUND)
 
-IF(CMAKE_COMPILER_IS_GNUCXX)
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wfatal-errors -O3 -DCG_BC -DQX_SPARSE_MV_MUL -D__BUILTIN_LINALG__ -Wno-sign-compare -msse4.2 -DUSE_OPENMP -Wno-unused-variable -Wno-reorder")
-ENDIF(CMAKE_COMPILER_IS_GNUCXX)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
This requires CMake 3.1 at least. I would hope no one actually uses 2.8 anymore, but then again, on CentOS machines this is still the default. Please make sure that changes made to CMakeLists files actually work on the minimal CMake version, or at least update the `CMAKE_MINIMUM_REQUIRED` line accordingly.

Full disclosure: I don't actually know if the current CMakeLists actually work with 2.8, but they seemed sufficiently simple for that to me.

Reply:
I've removed `CMAKE_CXX_STANDARD` and restored `-std=c++11` flag.